response_operations_ui/templates/collection-exercise.html

```diff
@@ -5,27 +5,96 @@
 <div class="container page__container">
   <div role="main" id="main" class="page__main">
     <br/>
-    <h1 class="jupiter">{{survey.shortName}} {{ce.exerciseRef}}</h1>
+    <h1 class="jupiter">{{survey.surveyRef}} {{survey.shortName}} {{ce.exerciseRef}}</h1>
 
     <div class="grid">
-      <div class="grid__col col-5@l"></div>
-      <div class="grid__col col-1@l"></div>
+      <div class="grid__col col-5@l">
```

Comment:
Think the left div should be 6 columns wide and the right div should be 5 columns wide